exercises/concept/chaitanas-colossal-coaster/.docs/introduction.md

```diff
@@ -99,7 +102,7 @@ If you want to remove all items from a `list`, use `list.clear()`. It does not t
 
 ## Reversing and reordering
 
-You can reverse the order of a list with the `list.reverse()` method.
+The `list.reverse()` method will reverse the order of elements.
```

Comment:
```suggestion
The `list.reverse()` method will reverse the order of elements in-place.
```

Reply:
Not sure why I can't auto-commit this.  Going to self-suggest.  Grr.